.github/workflows/deploy.sh

```diff
@@ -44,17 +45,24 @@ then
 ############### PRODUCTION ###############
 destination=prod
 ############### PRODUCTION ###############
+elif [ "$selection" = "4" ]
+then
+############### ACR PURGE ###############
+destination=""
+az acr run --cmd "acr purge --filter 'portal-ui:.*' --filter 'gef-ui:.*' --filter 'trade-finance-manager-ui:.*' --filter 'portal-api:.*' --filter 'trade-finance-manager-api:.*' --filter 'dtfs-central-api:.*' --filter 'reference-data-proxy:.*' --filter 'azure-function-number-generator:.*' --filter 'azure-function-acbs:.*' --ago 15d" --registry tfsdev /dev/null
```

Comment:
does this wipe all images from azure registry?

Reply:
Only ones which are older than `15` days.